lib/bozo/test_runners/nunit3.rb

```diff
@@ -57,6 +61,10 @@ def runner_args
         args << '--x86'
       end
 
+      unless @agents.blank?
```

Comment:
Also this method doesn't exist in the standard ruby library its a rails method  you want something like 

`agents && agents.empty?`